hazelcast/src/test/java/com/hazelcast/internal/serialization/impl/compact/CompactSerializationTest.java

```diff
@@ -278,6 +279,26 @@ public void testSerializingClassReflectively_withUnsupportedFieldType() {
                 .hasStackTraceContaining("which uses this class in its fields");
     }
 
+    @Test
+    public void testSerializingClassReflectively_withUnsupportedField_whenTheFieldHasExplicitSerializer() {
```

Comment:
Should we also add test for record containing such custom field?

Reply:
The tests for records are added